llvm/include/llvm/BinaryFormat/MachO.h

```diff
@@ -17,6 +17,7 @@
 #include "llvm/Support/DataTypes.h"
 #include "llvm/Support/Error.h"
 #include "llvm/Support/SwapByteOrder.h"
+#include "llvm/TargetParser/Triple.h"
```

Comment:
You can drop the forward-declare at l24 BTW;  I guess having Triple::ArchType be its own forward-declarable type would avoid this and other includes, but one problem at a time!